client/src/app/videos/+video-watch/video-watch.component.scss

```diff
@@ -66,6 +66,27 @@
       .video-info-channel {
         font-weight: $font-semibold;
         font-size: 15px;
+
+        a {
+          text-decoration: none;
```

Comment:
Maybe you could create a syndication component instead of duplicating HTML and CSS? :)